pinakes/main/migrations/0048_remove_notificationtype_main_notificationtype_n_type_unique_and_more.py

```diff
@@ -0,0 +1,39 @@
+# Generated by Django 4.0.2 on 2022-05-12 20:35
```

Comment:
@bzwei You might want to redo this migration since a recent migration change got merged